### PR TITLE
Added masking to Customer fields. Updated version, README, release notes

### PR DESCRIPTION
**Pull request by MikeField119**
Customer fields masked:
* firstName
* lastName
* idNumber
* address
* contactNumber

### CODE REVIEW

File: src/main/java/io/electrum/billpay/model/Customer.java [hunk 1]
```diff
@@ -2,6 +2,8 @@
```
Comment (TimElectrum) [inline src/main/java/io/electrum/billpay/model/Customer.java:2]: is there a reason we're not masking the address or contact number? I think they're also part of the POPI basket.

**Review by TimElectrum — CHANGES_REQUESTED** [review]
please update the README.md

File: src/main/java/io/electrum/billpay/model/Customer.java (hunk 1 above)
Comment (MikeField119) [reply src/main/java/io/electrum/billpay/model/Customer.java:2]: These fields weren't required for the Hello Paisa integration which is why they weren't included initially.
Will add.

File: README.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ To include the service interface into your maven project, include the below depe
 <dependency>
     <groupId>io.electrum</groupId>
     <artifactId>billpay-service-interface</artifactId>
-    <version>4.8.0</version>
+    <version>4.9.0</version>
```
Comment (MikeField119) [inline README.md:15]: @TimElectrum I've updated the version here in the README.md, what else needs to be updated in this file?

File: src/docs/devguide/hugo/site/content/release-notes.md [hunk 3]
```diff
@@ -1,5 +1,14 @@
 This page describes changes to the Billpay Service Interface implemented across different releases of the interface.
 
+## v4.9.0
+
+Released XX September/October 2020
+
+- Added masking to the following `Customer` model fields:
+  - `firstName`
+  - `lastName`
+  - `idNumber`
```
Comment (TimElectrum) [inline src/docs/devguide/hugo/site/content/release-notes.md:10]: and the address and contact number

**Review by TimElectrum — CHANGES_REQUESTED** [review]

File: src/docs/devguide/hugo/site/content/release-notes.md [hunk 4]
```diff
@@ -1,5 +1,16 @@
 This page describes changes to the Billpay Service Interface implemented across different releases of the interface.
 
+## v4.9.0
+
+Released XX September/October 2020
```
Comment (TimElectrum) [inline src/docs/devguide/hugo/site/content/release-notes.md:5]: just a reminder to update the release date

**Review by TimElectrum — APPROVED** [review]
just a reminder to update the release date

**Pull request merged**